executor/hash_table.go

```diff
@@ -32,18 +32,21 @@ import (
 )
 
 const (
+	// nolint:varcheck
 	// estCountMaxFactor defines the factor of estCountMax with maxChunkSize.
 	// estCountMax is maxChunkSize * estCountMaxFactor, the maximum threshold of estCount.
 	// if estCount is larger than estCountMax, set estCount to estCountMax.
 	// Set this threshold to prevent buildSideEstCount being too large and causing a performance and memory regression.
 	estCountMaxFactor = 10 * 1024
 
+	// nolint:varcheck
 	// estCountMinFactor defines the factor of estCountMin with maxChunkSize.
 	// estCountMin is maxChunkSize * estCountMinFactor, the minimum threshold of estCount.
 	// If estCount is smaller than estCountMin, set estCount to 0.
 	// Set this threshold to prevent buildSideEstCount being too small and causing a performance regression.
 	estCountMinFactor = 8
 
+	// nolint:varcheck
 	// estCountDivisor defines the divisor of buildSideEstCount.
 	// Set this divisor to prevent buildSideEstCount being too large and causing a performance regression.
 	estCountDivisor = 8
```

Comment:
Could we remove these constants? Then we don't need `nolint`.

Reply:
Sure, I've removed them.